### PR TITLE
Roc 4206 issue ccj claimant response tests

### PR DESCRIPTION
**Pull request by tharacka**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-4206

### Change description
Added Unit Tests only

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/app/claims/ccjModelConverter.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ import { MomentFactory } from 'shared/momentFactory'
 import { PartyType } from 'common/partyType'
 import { Individual } from 'claims/models/details/theirs/individual'
 import { Party } from 'claims/models/details/yours/party'
-import { ResponseType } from 'claims/models/response/responseType'
+// import { ResponseType } from 'claims/models/response/responseType'
```
Comment (timja) [inline src/main/app/claims/ccjModelConverter.ts]: ?

File: src/main/app/claims/ccjModelConverter.ts [hunk 2]
```diff
@@ -106,7 +95,7 @@ export class CCJModelConverter {
     const paymentDate = paymentIntention.paymentDate && paymentIntention.paymentDate.date.toMoment()
     const paymentOption = paymentIntention.paymentOption.option.value as PaymentOption
 
-    const alreadyPaidAmount: number = convertAlreadyPaidAmount(claim, claimantResponse)
+    const alreadyPaidAmount: number = claimantResponse.paidAmount.amount
```
Comment (timja) [inline src/main/app/claims/ccjModelConverter.ts]: why is this being removed?

File: src/test/app/claims/ccjModelConverter.ts [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+import { CCJModelConverter } from 'claims/ccjModelConverter'
+// Claim Data
+import { fullAdmissionWithImmediatePaymentData,
+  fullAdmissionWithPaymentBySetDateData,
+  fullAdmissionWithPaymentByInstalmentsData,
+  partialAdmissionWithImmediatePaymentCompanyData,
+  partialAdmissionWithPaymentBySetDateCompanyData,
+  partialAdmissionWithPaymentByInstalmentsCompanyData
+ } from 'test/data/entity/responseData'
+// Claimant-Response Draft and Data
```
Comment (timja) [inline src/test/app/claims/ccjModelConverter.ts]: please don't add comment in imports, IDEs will move them around when they are refactored and comments won't be updated

File: src/test/app/claims/ccjModelConverter.ts [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+import { DraftClaimantResponse } from 'claimant-response/draft/draftClaimantResponse'
+import { claimantResponseDraftWithPaymentMethodAccepted,
+  claimantResponseDraftWithPaymentMethodRejectedAndImmediatePaymentProposed,
+  claimantResponseDraftWithPaymentMethodRejectedAndPaymentBySetDateProposed
+} from 'test/data/draft/claimantResponseDraft'
+// CCJ Draft and Data
```
Comment (timja) [inline src/test/app/claims/ccjModelConverter.ts]: same here

File: src/test/app/claims/ccjModelConverter.ts [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+// import { CountyCourtJudgment } from 'claims/models/countyCourtJudgment'
```
Comment (timja) [inline src/test/app/claims/ccjModelConverter.ts]: please don't leave commented out code in

File: src/test/app/claims/ccjModelConverter.ts [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    const draft = prepareDraft(claimantResponseDraftWithPaymentMethodAccepted)
+
+    expect(convertObjectLiteralToJSON(CCJModelConverter.convertForIssue(claim, draft)))
+      .to.be.deep.equal(convertObjectLiteralToJSON(ccjIssueRequestPayByInstalments))
+  })
+
+})
+
+describe('CCJModelConverter - full admission by individual - claimant rejects', () => {
+  xit('should convert to CCJ - defendant pays immediately - claimant propose new immediate payment', () => {
```
Comment (timja) [inline src/test/app/claims/ccjModelConverter.ts]: I would recommend not committing skipped tests as they are easily forgotten about or missed,
if you need to commit or push with a failing test for review / help then you can add `--no-verify` to your commit and push commands

File: src/test/app/claims/ccjModelConverter.ts (hunk 3 above)
Comment (tharacka) [reply src/test/app/claims/ccjModelConverter.ts]: no problem, I will refrain from doing so. I only put them in to help me keep a track of the tasks I needed to do

File: src/main/app/claims/ccjModelConverter.ts (hunk 2 above)
Comment (timja) [reply src/main/app/claims/ccjModelConverter.ts]: makes sense, if you resolve the conflicts this diff will disappear then

File: src/test/app/claims/ccjModelConverter.ts (hunk 6 above)
Comment (tharacka) [reply src/test/app/claims/ccjModelConverter.ts]: Ok no problem, I was told to do so by Chris as the skipped tests cannot be tested at the moment, but wanted them in so they aren't forgotten about.

File: src/main/app/claims/ccjModelConverter.ts (hunk 1 above)
Comment (timja) [reply src/main/app/claims/ccjModelConverter.ts]: can you remove the import then please

File: src/test/app/claims/ccjModelConverter.ts (hunk 6 above)
Comment (tharacka) [reply src/test/app/claims/ccjModelConverter.ts]: Make sense, I will remove the tests for now

File: src/test/data/entity/ccjData.ts [hunk 7]
```diff
@@ -0,0 +1,54 @@
+import { MomentFactory } from 'shared/momentFactory'
+
+export const ccjIssueRequestPayImmediately = {
+  defendantDateOfBirth: MomentFactory.parse('1999-01-01').utc(true),
+  paidAmount: undefined,
```
Comment (timja) [inline src/test/data/entity/ccjData.ts]: probably don't need all these undefined properties set?

**Pull request merged**